kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/security/SecurityService.java

```diff
@@ -46,4 +44,24 @@ public interface SecurityService {
      */
     public boolean isDebugEnabled();
 
+    /**
+     * This method allows the load of the default production security policy
+     *
+     * @throws KuraException
+     *         in case an error is raised during the application of the default production security policy.
+     * @since 2.7
+     */
+
+    public void loadDefaultProductionSecurityPolicy() throws KuraException;
+
+    /**
+     * This method allows the load of the provided security policy
+     *
+     * @param securityPolicy
+     *         the security policy to be loaded
+     * @throws KuraException
+     *         in case an error is raised during the application of the security policy.
+     * @since 2.7
+     */
+    public void loadSecurityPolicy(String securityPolicy) throws KuraException;
```

Comment:
`load` in my opinion is too ambiguos: does it mean get, apply, or reload?

Reply:
> something like an `apply` method, but in this way is not clear that you have also to upload the policy

Maybe that is clear from the parameters. IMHO apply is better suited for both methods. We could describe the methods better in the javadoc